init.lua

```diff
@@ -696,12 +696,12 @@ function PaperWM:setWindowFullWidth()
     self:tileSpace(space)
 end
 
-function PaperWM:cycleWindowSize(direction)
+function PaperWM:cycleWindowSize(direction, cycleDirection)
```

Comment:
small nit, but we've been using camelCase for method names and snake_case for variable names in this project. Could you change to `cycle_direction`?